deploy-if-version-bump.sh

```diff
@@ -0,0 +1,13 @@
+# deploy-if-version-bump.sh
+#!/bin/sh
```

Comment:
This line states than your script should be using with the machine `sh`. This `sh` can be `zsh` on your machine, `bash` quite often, or some older and more basic shells in certain machines.

If your script uses `bash` features, and you declare `#!/bin/sh` here, it may not work on the latter machines.

To avoid this issue, it's more common to ask the machine to use `bash`. To do so, just replace this line by `#! /usr/bin/env bash`.